resources/definitions/fdmprinter.def.json

```diff
@@ -4919,6 +4919,17 @@
                     "enabled": "magic_spiralize",
                     "settable_per_mesh": false,
                     "settable_per_extruder": false
+                },
+                "relative_extrusion":
+                {
+                    "label": "Use Relative Extrusion",
+                    "description": "Use relative extrusion rather than absolute extrusion. Before, Cura always used absolute extrusion. Most printers can cope with either relative or absolute extrusion.",
```

Comment:
Ok maybe instead of saying `small differences` it should say `tiny deviations`.

Just to be clear: the deviation is as small as half a step of the stepper motor of the extruder. Depending on the printer that might be something which has a visible effect on the print quality.

Maybe you are right and we can leave that part out.
I don't want to be too negative, but I'm trying to list the advantages and disadvantages of this setting. If it only has advantages it should always be turned on of course!

You can tweak the exact wording a bit to your liking. It's your PR of course ;)